components/infobox/wikis/pubg/imfobox_map_custom.lua

```diff
@@ -0,0 +1,83 @@
+---
+-- @Liquipedia
+-- wiki=pubg
+-- page=Module:Infobox/Weapon/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Weapon = require('Module:Infobox/Weapon')
+local Class = require('Module:Class')
+local String = require('Module:String')
+local Injector = require('Module:Infobox/Widget/Injector')
+local Cell = require('Module:Infobox/Widget/Cell')
+local Title = require('Module:Infobox/Widget/Title')
+local Center = require('Module:Infobox/Widget/Center')
+local PageLink = require('Module:Page')
+
+local CustomWeapon = Class.new()
+local CustomInjector = Class.new(Injector)
+
+local _weapon
+local _args
+
+function CustomWeapon.run(frame)
+	local weapon = Weapon(frame)
+	_weapon = weapon
+	_args = _weapon.args
+	weapon.createWidgetInjector = CustomWeapon.createWidgetInjector
+	weapon.addToLpdb = CustomWeapon.addToLpdb
+	return weapon:createInfobox(frame)
+end
+
+function CustomWeapon:createWidgetInjector()
+	return CustomInjector()
+end
+
+function CustomInjector:addCustomCells(widgets)
+	local args = _args
+	table.insert(widgets, Cell{
+		name = 'Ammo Type',
+		content = {args.ammo_type}
```

Comment:
```suggestion
		content = {args.ammotype}
```

Reply:
current module just uses `ammo` for this

Reply:
Which module are you referring to to uses ``ammo``

Reply:
template i mean, sry

Reply:
Asked for the feedback of the users. ``ammotype`` > ``ammo``
![image](https://user-images.githubusercontent.com/58013431/151669932-6acb89e1-9cb7-4b2c-a7e6-83d2e4dd648c.png)

Reply:
Asked for the feedback of the users. ``ammotype`` > ``ammo``
![image](https://user-images.githubusercontent.com/58013431/151669932-6acb89e1-9cb7-4b2c-a7e6-83d2e4dd648c.png)

Reply:
might need bot jobs then, so keep that in mind